front/app/utils/actionDescriptors/messages.ts

```diff
@@ -0,0 +1,38 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  // Defaults
+  votingNotSignedIn: {
+    id: 'app.utils.participationMethodConfig.voting.votingNotSignedIn',
+    defaultMessage: 'You must login or register to vote.',
```

Comment:
Isn't there a typo here and in some other places as "log in" is a verb?

```suggestion
    defaultMessage: 'You must log in or register to vote.',
```

Reply:
What copies did we use before for the errors in this file?

Reply:
There was no copy, so people would just be faced with a button that looked clickable but would not click and had no message